Documentation/operations/performance/tuning.rst

```diff
@@ -38,12 +40,11 @@ look for the line reporting the status for "Host routing".
 Bypass iptables Connection Tracking
 ===================================
 
-Even when routing is performed by Cilium using eBPF host-routing, network
-packets still traverse the regular network stack in the network namespace
-of the container and iptables can again add significant cost. This traversal
-cost can be minimized by disabling the connection tracking requirement
-for packets forwarded by Cilium, thus bypassing the iptables connection
-tracker.
+For the case when eBPF Host-Routing cannot be used and thus network packets
+still need to traverse the regular network stack in the host namespace, then
+iptables can again add significant cost. This traversal cost can be minimized
+by disabling the connection tracking requirement for packets forwarded by Cilium,
```

Comment:
> for packets forwarded by Cilium

nit: to keep this in line with the documentation of the `--install-no-conntrack-iptables-rules` option ("Install Iptables rules to skip netfilter connection tracking on all pod traffic") maybe we can change it to "for all pod traffic"? (as also hostns -> pod traffic is matched by these rules, but this traffic is not entirely forwarded by Cilium)

Reply:
I've changed into `This traversal cost can be minimized by disabling the connection tracking requirement for all Pod traffic, thus bypassing the iptables connection tracker.`